core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/first.kt

```diff
@@ -62,26 +173,209 @@ public inline fun <T> DataFrame<T>.firstOrNull(predicate: RowFilter<T>): DataRow
 
 // region GroupBy
 
+/**
+ * Selects the first row from each group of the given [GroupBy]
+ * and returns a [ReducedGroupBy] containing these rows
+ * (one row per group, each row is the first row in its group).
+ *
+ * ### Example
+ * ```kotlin
+ * // Select the first employee from each group formed by the job title
+ * employees.groupBy { jobTitle }.first()
```

Comment:
A `ReducedGroupBy` is rarely the end. It may make sense to add `concat()`, `into()`, or `values()` after it